tests/filters/test_background_color.py

```diff
@@ -36,3 +36,24 @@ def config_context(context):
 
         ssim = self.get_ssim(image, expected)
         expect(ssim).to_be_greater_than(0.97)
+
+    @gen_test
+    async def test_background_color_filter_with_auto_color(self):
+        def config_context(context):
+            context.request.fit_in = True
+            context.request.width = 300
+            context.request.height = 300
+
+        image = await self.get_filtered(
+            "PNG_transparency_demonstration_1.png",
+            "thumbor.filters.background_color",
+            "background_color(auto)",
+            config_context=config_context,
```

Comment:
In order to decrease the amount of repeated code, would you consider creating a new `config_context` method and having this line — and the respective one of the test above — read the following?
```suggestion
            config_context=self.config_context,
```